lib/tapioca/helpers/rbi_files_helper.rb

```diff
@@ -150,21 +150,19 @@ def validate_rbi_files(command:, gem_dir:, dsl_dir:, auto_strictness:, gems: [],
 
     private
 
-    sig { params(index: RBI::Index, files: T::Array[String]).void }
-    def parse_and_index_files(index, files)
-      files.each do |file|
-        parse_and_index_file(index, file)
-      end
-    end
+    sig { params(index: RBI::Index, files: T::Array[String], number_of_workers: T.nilable(Integer)).void }
+    def parse_and_index_files(index, files, number_of_workers:)
```

Comment:
Should we set the default value of `number_of_workers` to 1 to avoid having to pass it for sequential tasks?

Reply:
This is a private low-level command, if you don't want to deal with the details you should call the public `index_rbi` and `index_rbis` methods.